Statistics

```diff
@@ -0,0 +1,104 @@
+def most_sold():
+    """We will need a list representing the number of times a product has been bought. Hence, 
+    we will need to implement a function where the user can register the sales. """
+    
+    return ('We should return the product with the hightest sales according to the table above')
+ 
+def least_sold(): 
+    """We will need a list representing the number of times a product has been bought. Hence, 
```

Comment:
If you need a list, why isn't a parameter here?

Reply:
I'll put the appropiate parameters for the functions according to the information within the docstrings.

Reply:
did you?